gluten-core/src/main/scala/io/glutenproject/extension/ColumnarOverrides.scala

```diff
@@ -786,6 +786,7 @@ case class ColumnarOverrideRules(session: SparkSession)
       List(
         (spark: SparkSession) => PlanOneRowRelation(spark),
         (_: SparkSession) => FallbackEmptySchemaRelation(),
+        (spark: SparkSession) => RemoveKnownFloatingPointNormalized(spark),
```

Comment:
We should apply this rule after `AddTransformHintRule` and check that we apply this rule only if we support transform aggregate